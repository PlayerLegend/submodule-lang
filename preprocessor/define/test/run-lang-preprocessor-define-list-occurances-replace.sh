#!/bin/sh

cat src/lang/preprocessor/define/test/asdf.replace | $DEBUG_PROGRAM test/lang-preprocessor-define-list-occurances-replace asdf
