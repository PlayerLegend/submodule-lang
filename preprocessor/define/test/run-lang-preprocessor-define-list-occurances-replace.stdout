2 22
3 7
3 45
3 77
6 7
6 27
(
	(
		lit [this]
		lit [is]
		lit [a]
		lit [list]
	)
	(
		lit [this]
		lit [is]
		lit [a]
		lit [list]
		lit [with]
		(
			lit [bcle]
		)
	)
	(
		lit [this]
		(
			lit [bcle]
		)
		lit [is]
		lit [in]
		lit [a]
		lit [new]
		lit [list]
		(
			lit [with]
			lit [some]
			lit [more]
			(
				lit [bcle]
			)
		)
		(
			lit [nested]
			lit [within]
		)
		(
			lit [but]
			lit [only]
			(
				lit [bcle]
			)
			lit [intermittently]
		)
	)
	(
	)
	(
		lit [more]
		lit [lists]
	)
	(
		lit [more]
		(
			lit [bcle]
		)
		lit [lists]
		(
			lit [within]
			(
				(
					lit [bcle]
				)
			)
			lit [lists]
		)
	)
)
